packages/block-library/src/navigation-menu/block-colors-selector.js

```diff
@@ -25,20 +25,20 @@ const ColorSelectorSVGIcon = () => (
  * @return {*} React Icon component.
  */
 const ColorSelectorIcon = ( { textColor, textColorValue } ) => {
-	const iconStyle = {};
-
-	if ( textColorValue ) {
-		iconStyle.color = textColorValue;
-	}
-
-	const iconClasses = classnames( 'block-library-colors-selector__state-selection', {
-		'has-text-color': textColor && textColor.color,
-		[ textColor.class ]: textColor && textColor.class,
-	} );
+	const iconClasses = classnames(
+		'block-library-colors-selector__state-selection',
+		'editor-styles-wrapper', {
```

Comment:
Should the brace be on a new line?

Reply:
I did it on purpose since it looks clearer to reading, to me. Glad to change it if you consider it does should be changed.